code/web/src/modules/crate/Item.js

```diff
@@ -35,7 +35,7 @@ class Item extends PureComponent {
 		})
 		
 		if (!this.props.stylePref.style) {
-			this.props.messageShow('Loading Style Survey...')
+			// this.props.messageShow('Loading Style Survey...')
 			this.props.history.push(surveyRoutes.survey.path)
```

Comment:
Couldn't get loading message to disappear, so taking out for now. Better to be simple & good, then hacky.